<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Login</class>
 <widget class="QMainWindow" name="Login">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>700</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>900</width>
    <height>700</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">background-color: #E6E6E6</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QWidget" name="login" native="true">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QWidget" name="widget_4" native="true">
         <layout class="QGridLayout" name="gridLayout_4">
          <item row="0" column="0">
           <widget class="QWidget" name="popup" native="true">
            <property name="maximumSize">
             <size>
              <width>315</width>
              <height>32</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Yu Gothic UI Semibold</family>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgb(133, 148, 133);
border-radius: 10px</string>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QLineEdit" name="lineEdit">
               <property name="font">
                <font>
                 <family>Yu Gothic UI Semibold</family>
                 <pointsize>11</pointsize>
                 <weight>75</weight>
                 <bold>true</bold>
                </font>
               </property>
               <property name="styleSheet">
                <string notr="true">color: white;</string>
               </property>
               <property name="text">
                <string>usuário ou senha incorreta!</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
               <property name="readOnly">
                <bool>true</bool>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="widget" native="true">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>315</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <layout class="QGridLayout" name="gridLayout_3">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <property name="spacing">
           <number>0</number>
          </property>
          <item row="0" column="0">
           <widget class="QWidget" name="container_login" native="true">
            <property name="maximumSize">
             <size>
              <width>315</width>
              <height>337</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Yu Gothic UI Semibold</family>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="cursor">
             <cursorShape>ArrowCursor</cursorShape>
            </property>
            <property name="styleSheet">
             <string notr="true">QWidget {
	background-color: rgba(63, 66, 49, 100);
	border-radius: 25px;
}

QLineEdit {
	background-color: rgb(199, 199, 199);
}

QLineEdit::hover {
	
	background-color: rgb(230, 230, 230);
}</string>
            </property>
            <widget class="QLabel" name="icon_user">
             <property name="geometry">
              <rect>
               <x>40</x>
               <y>60</y>
               <width>40</width>
               <height>40</height>
              </rect>
             </property>
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="pixmap">
              <pixmap resource="../../../../git/Unit-controle-financeiro/resources/autoFiles/qrc.qrc">:/login/img/login/user.png</pixmap>
             </property>
            </widget>
            <widget class="QLabel" name="icon_padlock">
             <property name="geometry">
              <rect>
               <x>40</x>
               <y>130</y>
               <width>40</width>
               <height>40</height>
              </rect>
             </property>
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="pixmap">
              <pixmap resource="../../../../git/Unit-controle-financeiro/resources/autoFiles/qrc.qrc">:/login/img/login/padlock.png</pixmap>
             </property>
            </widget>
            <widget class="QLineEdit" name="login_entry_password">
             <property name="geometry">
              <rect>
               <x>80</x>
               <y>130</y>
               <width>191</width>
               <height>41</height>
              </rect>
             </property>
             <property name="cursor">
              <cursorShape>IBeamCursor</cursorShape>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="maxLength">
              <number>15</number>
             </property>
             <property name="frame">
              <bool>true</bool>
             </property>
             <property name="echoMode">
              <enum>QLineEdit::Password</enum>
             </property>
             <property name="cursorPosition">
              <number>0</number>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
             <property name="dragEnabled">
              <bool>false</bool>
             </property>
             <property name="placeholderText">
              <string>senha</string>
             </property>
             <property name="cursorMoveStyle">
              <enum>Qt::LogicalMoveStyle</enum>
             </property>
             <property name="clearButtonEnabled">
              <bool>false</bool>
             </property>
            </widget>
            <widget class="QLineEdit" name="login_entry_user">
             <property name="geometry">
              <rect>
               <x>80</x>
               <y>60</y>
               <width>191</width>
               <height>41</height>
              </rect>
             </property>
             <property name="cursor">
              <cursorShape>IBeamCursor</cursorShape>
             </property>
             <property name="inputMethodHints">
              <set>Qt::ImhNone</set>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="maxLength">
              <number>15</number>
             </property>
             <property name="frame">
              <bool>true</bool>
             </property>
             <property name="echoMode">
              <enum>QLineEdit::Normal</enum>
             </property>
             <property name="cursorPosition">
              <number>0</number>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
             <property name="placeholderText">
              <string>usuário</string>
             </property>
             <property name="cursorMoveStyle">
              <enum>Qt::LogicalMoveStyle</enum>
             </property>
            </widget>
            <widget class="QCheckBox" name="login_check_lembrar">
             <property name="geometry">
              <rect>
               <x>100</x>
               <y>200</y>
               <width>111</width>
               <height>17</height>
              </rect>
             </property>
             <property name="font">
              <font>
               <family>Yu Gothic UI Semibold</family>
               <pointsize>9</pointsize>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="cursor">
              <cursorShape>ArrowCursor</cursorShape>
             </property>
             <property name="styleSheet">
              <string notr="true">QCheckBox {
	color: white;
	background-color: rgba(255, 255, 255, 0);
}

QCheckBox::hover {
	
	color: rgb(227, 227, 227);
}</string>
             </property>
             <property name="text">
              <string>lembrar de mim</string>
             </property>
            </widget>
            <widget class="QPushButton" name="btn_login">
             <property name="geometry">
              <rect>
               <x>50</x>
               <y>260</y>
               <width>211</width>
               <height>31</height>
              </rect>
             </property>
             <property name="font">
              <font>
               <family>Yu Gothic UI Semibold</family>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">QPushButton {
	color: rgb(255, 255, 255);
	border-radius: 10px;
	background-color: rgb(110, 125, 111);
}

QPushButton::hover {
	
	background-color: rgb(142, 161, 143);
}</string>
             </property>
             <property name="text">
              <string>LOGIN</string>
             </property>
            </widget>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="logos" native="true">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>113</height>
       </size>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QFrame" name="logo_escola">
         <property name="minimumSize">
          <size>
           <width>94</width>
           <height>0</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">border-radius: 47px;</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QLabel" name="png_logo">
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="pixmap">
             <pixmap resource="../../../../git/Unit-controle-financeiro/resources/autoFiles/qrc.qrc">:/login/img/login/logo_pedras_vivas.png</pixmap>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QWidget" name="widget_3" native="true">
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="0">
           <widget class="QWidget" name="widget_2" native="true">
            <property name="minimumSize">
             <size>
              <width>96</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>0</width>
              <height>50</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgb(110, 125, 111);
border-radius: 10px;
</string>
            </property>
            <layout class="QGridLayout" name="gridLayout">
             <item row="0" column="0">
              <widget class="QLabel" name="logo_unit">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>0</height>
                </size>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="pixmap">
                <pixmap resource="../../../../git/Unit-controle-financeiro/resources/autoFiles/qrc.qrc">:/login/img/login/logo_unit.png</pixmap>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <action name="actionlembrar">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Lembre de mim</string>
   </property>
  </action>
  <action name="actionversao">
   <property name="text">
    <string>versão</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>lineEdit</tabstop>
  <tabstop>login_entry_user</tabstop>
  <tabstop>login_entry_password</tabstop>
  <tabstop>login_check_lembrar</tabstop>
  <tabstop>btn_login</tabstop>
 </tabstops>
 <resources>
  <include location="../../../../git/Unit-controle-financeiro/resources/autoFiles/qrc.qrc"/>
 </resources>
 <connections/>
</ui>
